.gitconfig

```diff
@@ -0,0 +1,10 @@
+[core]
```

Comment:
Interested in your decision to include your .gitconfig file. Can you explain a little bit about what challenges or opportunities this approach might have in a team environment?